### ISSUE TITLE
can't make ieds

### ISSUE THREAD

**Issue by newfren** [tgstation/tgstation#82295]
Reporting client version: 515.1634

## Issue Summary
cannot craft ieds

## Round ID:
[226311](https://scrubby.melonmesa.com/round/226311)

## Testmerges:
- [[Testmerge Only] [No React Randys] Removes the paralyse from shoving someone while they are knocked down](https://www.github.com/tgstation/tgstation/pull/81886)
- [Makes lua file upload work with bigger files and standardizes a few file|null](https://www.github.com/tgstation/tgstation/pull/82202)

## Reproduction:
![image](https://github.com/tgstation/tgstation/assets/12635164/b99f5919-5193-422a-b675-92c7d2c6489e)
(I had a welder and a screwdriver in my belt)
(that beaker has 20u welding fuel)
get a timer, cable coils, a pipe, welding fuel, a screwdriver, and a welding tool
attempt to craft improvised explosive
button is greyed out

**Comment by DaCoolBoss** [comment]
Does the problem persist if you add an uncrushed soda can to this list of items?

**Comment by Jackal-boop** [comment]
IEDs were replaced weren't they

**Comment by Profakos** [comment]
Can not reproduce with a hand crafted pipe, however, I could reproduce by wrenching up an air supply pipe, due to the premade pipe being  an instance of `/obj/item/pipe/quaternary `, and the handmade one is `/obj/item/pipe/quaternary/pipe/crafted `. The recipe asks for `obj/item/pipe/quaternary/pipe`. Where did you get that green pipe in the example above?

**Comment by newfren** [comment]
used an rcd to produce it, then unwrenched it

**Comment by Profakos** [comment]
Yeah that deconstructs into `/obj/item/pipe/quaternary` instead of `/obj/item/pipe/quaternary/pipe`, which causes the issue.

**Comment by Profakos** [comment]
The creator of the new IEDs claims this is intentional, and did not want it to work with non hand crafted pipes. The recipe should get a note or something, making it clearer if the type in the recipe is unchanged.

**Comment by JohnFulpWillard** [comment]
Dumb as hell intention, two items that act and look the exact same but recipe requires one. Someone kill it please.

**Comment by iprice** [comment]
Welp, since I duped the bug report I'm gonna second this as dumb.  Even with admin tooling to check type paths and a copy of the code base to figure out the exact subtype of various pipes this is non intuitive and feels broken rather than intended, even knowing the type it requires doesn't tell you how to obtain one.  I've no idea how a player is supposed to figure this out.
